x/compliance/keeper/msg_server_certify_model.go

```diff
@@ -76,6 +76,10 @@ func (k msgServer) CertifyModel(goCtx context.Context, msg *types.MsgCertifyMode
 	} else {
 		// There is no compliance record yet. So certification will be tracked on ledger.
 
```

Comment:
Need to think what to do in case we certify existing revoked or provisioned certification info. Should we update all new fields if they have been already set for the existing value...
I think we need to update them with new values if new values are set. Otherwise keep the old values.

Reply:
Done! ✅